client/src/pages/ManageBookings/Booking.tsx

```diff
@@ -0,0 +1,47 @@
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import SettingsIcon from '@material-ui/icons/Settings';
+
+import useStyles from './useStyles';
+
+interface BookingProps {
+  outlined?: boolean;
+  accepted?: boolean;
+  large?: boolean;

```

Comment:
Same as here, would this ever be null or undefined?

Reply:
These ones I would like to keep. It makes the Booking object act more like a MUI component, where we can include just the word "outlined", "accepted", or "large" to change the behaviour of the booking. As I have it now, these are sometimes null.